feast-ui/rockcraft.yaml

```diff
@@ -35,15 +34,14 @@ parts:
     source-tag: v0.49.0
     plugin: nil
     stage-packages:
-      - python3.10
-      - python3.10-venv
+      - python3.12
+      - python3.12-venv
       - tzdata
     build-snaps:
       - node/16/stable
     override-build: |
-      # Install Feast and PostgreSQL extra
-      pip3 install feast==0.49.0
-      pip3 install feast[postgres]==0.49.0
+      # Install Feast with PostgreSQL support, bypassing PEP 668 restrictions
```

Comment:
question: what restrictions were we seeing here? is it safe to bypass those?

Reply:
Thats was just added to the 24.04 its not a problem for the container as the only purpose of the container is to run that pebble process ... its there only for os things.